code/datums/status_effects/debuffs/heart_attack.dm

```diff
@@ -0,0 +1,164 @@
+///Stage two of the heart attack.
+#define ATTACK_STAGE_TWO 100
+///Stage three of the heart attack.
+#define ATTACK_STAGE_THREE 70
+///Stage four of the heart attack.
+#define ATTACK_STAGE_FOUR 35
+///If we reduce heart damage enough, it will recover on its own.
+#define ATTACK_CURE_THRESHOLD 160
+///What is the max oxyloss we're willing to deal, to prevent people from passing out early.
+#define OXYLOSS_MAXIMUM 40
+
+/datum/status_effect/heart_attack
+	id = "heart_attack"
+	status_type = STATUS_EFFECT_UNIQUE
+	remove_on_fullheal = TRUE
+	alert_type = null
+	///A timer that ticks down until the heart fully stops
+	var/time_until_stoppage = (150 SECONDS)
+	///Does the victim hear their own heartbeat?
+	var/sound = FALSE
```

Comment:
It seems like this value is sometimes set to FALSE but never set to true, and never read?